src/Dumplie/Domain/Customer/Checkout.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Domain\Customer;
+
+final class Checkout
+{
+    /**
+     * @var CartId
+     */
+    private $cartId;
+
+    /**
+     * @var Address
+     */
+    private $billingAddress;
+
+    /**
+     * @var Address
+     */
+    private $shippingAddress;
+
+    /**
+     * @param CartId $cartId
+     * @param Address $billingAddress
+     */
+    public function __construct(CartId $cartId, Address $billingAddress)
```

Comment:
It's good that _billing address_ is required, but it would be nice to be able to change after creation.